src/status_im2/contexts/chat/messages/view.cljs

```diff
@@ -58,7 +58,8 @@
 
 (defn chat-render []
   (let [;;we want to react only on these fields, do not use full chat map here
-        {:keys [chat-id show-input?] :as chat} (rf/sub [:chats/current-chat-chat-view])
+        show-input? (not (rf/sub [:group-chat/removed-from-current-chat?]))
```

Comment:
this change looks suspicious , could you elaborate?

Reply:
We hide the input a member has been removed. `show-input` value from `:chats/current-chat-chat-view` was always returning true even when the member is removed. So I used this other subscription `:group-chat/removed-from-current-chat?]`